src/js/views/loading_view.js.coffee

```diff
@@ -0,0 +1,13 @@
+define [
```

Comment:
@mwise Good call. I went ahead and renamed any `.js.coffee` file to be just `.coffee`.